site/source/docs/debugging/Sanitizers.rst

```diff
@@ -30,19 +30,16 @@ Catching Null Dereference
 -------------------------
 
 By default, with Emscripten, dereferencing a null pointer does not immediately
-cause a segmentation fault, unlike traditional platforms. Instead, it checks
-a magic cookie stored at address 0 at the end of the program execution.
+cause a segmentation fault, unlike traditional platforms, as 0 is just a normal
+address in a WebAssembly Memory. 0 is also a normal location in a
```

Comment:
This could probably go either way, but I was surprised by the uppercase "Memory" here.

```suggestion
address in a WebAssembly memory. 0 is also a normal location in a
```

Reply:
Hmm, `Memory` with a capital is the name of the thing in the JS API, but yeah, maybe that's not reason enough to do it.